pubspec.yaml

```diff
@@ -50,7 +50,7 @@ dependencies:
   http_parser: ^4.0.2
   image_picker: ^1.0.0
   json_annotation: ^4.9.0
-  mime: ^1.0.5
+  mime: ^2.0.0
```

Comment:
The breaking change here is in `extensionFromMime`.

Looks like we don't use that method:
```
$ git grep extensionFromMime
$
```
so doesn't affect us.